tests/test_field.py

```diff
@@ -29,6 +29,24 @@ def hello(self, info, id: int) -> str:
     assert hello.field.args["id"].type.of_type.name == "Int"
 
 
+def test_field_default_arguments_are_optional():
+    @strawberry.field
+    def hello(self, info, test: int, id: int = 1, asdf: str = "hello") -> str:
```

Comment:
well done! I'd also add a test where the default value is `None`, just in case :D

Reply:
Good call!